### PR TITLE
New Nexus Kiln EA

### PR DESCRIPTION
**Pull request by mxiao-cll**
## Description

Calculates Nexus vault value + Kiln Fees

Kiln Fees:
- First query graphQL to get all the validatorIds
- Go to contracts to get all fee address from validatorIds
- Call eth-balance to sum all the eths
- Call contract to find the discount % and apply to all the eths

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `adapter-secrets` configuration file or update the [soak testing blacklist](/packages/scripts/src/get-changed-adapters/soakTestBlacklist.ts).
- [ ] If a new adapter was made, or a new endpoint was added, update the `test-payload.json` file with relevant requests.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Jira.
- [ ] This is related to a maximum of one Jira story or GitHub issue.
- [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types).
- [ ] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.

### CODE REVIEW

File: packages/composites/nexus-kiln/src/abi/FundValueCalculatorRouter.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  },
+  {
+    "inputs": [{ "internalType": "address", "name": "_vaultProxy", "type": "address" }],
+    "name": "calcGav",
+    "outputs": [
+      { "internalType": "address", "name": "denominationAsset_", "type": "address" },
+      { "internalType": "uint256", "name": "gav_", "type": "uint256" }
+    ],
+    "stateMutability": "view",
+    "note": "THIS HAS BEEN MODIFIED TO BE A VIEW FUNCTION",
```
Comment (alecgard) [inline packages/composites/nexus-kiln/src/abi/FundValueCalculatorRouter.json:15]: Is this note from the original ABI, or something we have changed?

Comment (mxiao-cll) [reply packages/composites/nexus-kiln/src/abi/FundValueCalculatorRouter.json:15]: This is something we have changed

Comment (alecgard) [reply packages/composites/nexus-kiln/src/abi/FundValueCalculatorRouter.json:15]: Why did we need this change?
If it is just to be able to call this function without expecting state changes, can we use `callStatic`? https://docs.ethers.org/v5/api/contract/contract/#contract-callStatic

Comment (mxiao-cll) [reply packages/composites/nexus-kiln/src/abi/FundValueCalculatorRouter.json]: Updated

**Review by alecgard — APPROVED** [review]

**Pull request merged**